libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -122,7 +111,15 @@ bool ASTJsonConverter::visit(ImportDirective const& _node)
 
 bool ASTJsonConverter::visit(ContractDefinition const& _node)
 {
-	addJsonNode(_node, "ContractDefinition", { make_pair("name", _node.name()) }, true);
+	Json::Value linearizedBaseContracts(Json::arrayValue);
+	for (auto const& baseContract: _node.annotation().linearizedBaseContracts)
+		linearizedBaseContracts.append(reinterpret_cast<Json::UInt64>(baseContract));
```

Comment:
It is a pointer, those are used as opaque identifiers for the nodes, they can be used to reference the nodes from somewhere else.